packages/paste-core/components/date-picker/__tests__/index.spec.tsx

```diff
@@ -84,6 +85,97 @@ describe('DatePicker', () => {
     });
   });
 
+  describe('Customization', () => {
+    const initialProps = {
+      id: 'datepicker',
+      name: 'dp',
+      defaultValue: '2020-03-21',
+      'data-testid': 'datepickerid',
+    };
+    it('should add the element dom attibute', () => {
+      const {container} = render(<DefaultDatePicker {...initialProps} />);
+      expect(container.querySelector('[data-paste-element="DATEPICKER"]')).toBeInTheDocument();
+      expect(screen.getByTestId('datepickerid').getAttribute('data-paste-element')).toEqual('DATEPICKER_ELEMENT');
```

Comment:
Why is `data-paste-element` both `DATEPICKER` and `DATEPICKER_ELEMENT`?  I'm confused where the `_ELEMENT` suffix comes from since I can't find where you're adding it in `DatePicker.tsx`

Reply:
Yeah, so it's passed down to `Input`, and inside `Input` is[ where it gets split out.](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/input/src/Input.tsx#L95) 

By default Input outputs `INPUT` and `INPUT_ELEMENT`, but if you override it with an element name, you get a custom name